src/Domain/InterfaceComputeTags.hpp

```diff
@@ -206,15 +206,18 @@ struct InterfaceCompute<DirectionsTag, Direction<VolumeDim>>
       Tags::Interface<DirectionsTag, Direction<VolumeDim>> {
   static std::string name() noexcept { return "Interface"; }
   using tag = Direction<VolumeDim>;
+  using return_type =
+      std::unordered_map<::Direction<VolumeDim>, ::Direction<VolumeDim>>;
+  using argument_tags = tmpl::list<DirectionsTag>;
   static constexpr auto function(
+      const gsl::not_null<
+          std::unordered_map<::Direction<VolumeDim>, ::Direction<VolumeDim>>*>
+          result,
       const std::unordered_set<::Direction<VolumeDim>>& directions) noexcept {
-    std::unordered_map<::Direction<VolumeDim>, ::Direction<VolumeDim>> result;
     for (const auto& d : directions) {
-      result.emplace(d, d);
+      result->emplace(d, d);
```

Comment:
`emplace` will not overwrite an existing element, so please use `insert_or_assign`